pkgs/applications/misc/veracrypt/default.nix

```diff
@@ -4,12 +4,12 @@ with stdenv.lib;
 
 stdenv.mkDerivation rec {
   pname = "veracrypt";
-  version = "1.23";
-  minorVersion = "-Hotfix-2";
+  version = "1.24";
+  hotfix = "-Hotfix1";
```

Comment:
This won't let `nix` know there is an update if an Hotfix2 gets released because it will be still be version 1.24
So we should define `version = "1.24-Hotfix1";` instead

Reply:
Addressed in cffaa114cac2a2d0e448d8ef81041b4f644903cf